<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryAllocationAlgorithmWindow</class>
 <widget class="QMainWindow" name="MemoryAllocationAlgorithmWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory Allocation Algorithm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="RadioButton_FirstFit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>First Fit</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="RadioButton_BestFit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Best Fit</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="RadioButton_NextFit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Fit</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Algorithms</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Confirm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Back">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
